### PR TITLE
Improvement/2022/home search clear icon

### PR DESCRIPTION
**Pull request by Tolik170**
## Description

Add an icon which clear search field in home page

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| before|  after |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/search-bar/ClearIcon.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    const clearInputIcon = props.clearInputIcon;
+    const clearIconBlock = props.clearIconBlock;
+
+    const clearSearchInput = () => {
+        props.setSearchValue('');
+    }
+    
+    return (
+        <>
```
Comment (maarekua) [inline src/containers/search-bar/ClearIcon.js:26]: Redundant <>.

File: src/containers/search-bar/search-bar.js [hunk 2]
```diff
@@ -71,7 +72,7 @@ const SearchBar = ({
 
   return (
     <div className={mainClass}>
-      <SearchIcon />
+      <SearchIcon searchInputIcon={styles.searchInputIcon} />
```
Comment (maarekua) [inline src/containers/search-bar/search-bar.js:75]: https://mui.com/material-ui/api/input-adornment/#main-content

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/search-bar/ClearIcon.js (hunk 1 above)
Comment (Tolik170) [reply src/containers/search-bar/ClearIcon.js]: reworked.

File: src/containers/search-bar/search-bar.js (hunk 2 above)
Comment (Tolik170) [reply src/containers/search-bar/search-bar.js]: reworked.

**Pull request closed without merging**